src/main/dialogs/quickSetStrip/qssDialog.js

```diff
@@ -153,10 +171,111 @@ fluid.defaults("gpii.app.qss", {
             args: [
                 "{arguments}.0" // state
             ]
+        },
+        getQssHorizontalMargin: {
+            funcName: "gpii.app.qss.getQssHorizontalMargin",
+            args: ["{that}", "{qssWrapper}.qssWidget"]
+        },
+        getExtendedWidth: {
+            funcName: "gpii.app.qss.getExtendedWidth",
+            args: ["{that}"]
+        },
+        fitToScreen: {
+            funcName: "gpii.app.qss.fitToScreen",
+            args: ["{that}"]
         }
     }
 });
 
+
+
+/**
+ * Computes the total width of all of the QSS buttons, based on their sizes inside
+ * the BrowserWindow.
+ * @param {Component} that - The `gpii.app.qss` instance
+ * @param {Object[]} buttons - The list of QSS buttons
+ * @return {Number} - The total scaled size of the QSS's buttons
+ */
+gpii.app.qss.computeQssButtonsWidth = function (that, buttons) {
```

Comment:
Would be clearer here to pass in that.options and that.model separately rather than the whole component, to emphasise that this is a pure calculation function that doesn't have side effects on the component.